lib/grpc/GrpcService.ts

```diff
@@ -134,6 +134,11 @@ const createPlaceOrderEvent = (e: PlaceOrderEvent) => {
   return placeOrderEvent;
 };
 
+const getSwapFailureReason = (reason: SwapFailureReason) => {
```

Comment:
This seems a bit unnecessary to me, elsewhere we do similar ways to convert enums values to their string names without a dedicated function. Any objection to just using `SwapFailureReason[deal.failureReason]` in place of this method? Seems like less code.